### PR TITLE
feat(ci): :sparkles: Check MSRV in CI

### PR DESCRIPTION
**Pull request by zarik5**

### CODE REVIEW

**Review by The-personified-devil — APPROVED** [review]

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -158,3 +158,41 @@ jobs:
...
+          toolchain: stable
+          override: true
+      - uses: Swatinem/rust-cache@v1
+
+      - name: Build and install dependencies
+        env:
+          RUST_BACKTRACE: 1
+        run: |
+          sudo apt update
+          sudo apt install build-essential pkg-config nasm libva-dev libdrm-dev libvulkan-dev libx264-dev libx265-dev cmake libasound2-dev libjack-jackd2-dev libxrandr-dev libunwind-dev libgtk-3-dev
```
Comment (Meister1593) [inline .github/workflows/rust.yml:195]: do we really need libx264-dev anymore?

Comment (Meister1593) [reply .github/workflows/rust.yml:195]: ah... `cargo xtask prepare-deps --platform linux --no-nvidia` is not gpl, i guess so

Comment (The-personified-devil) [reply .github/workflows/rust.yml:195]: libx264 is used directly for the software encoder, so I'd assume so

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -158,3 +158,41 @@ jobs:
...
+
+      - run: |
+          cargo xtask prepare-deps --platform windows
+          cargo xtask check-msrv
+
+  check-msrv-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
```
Comment (Meister1593) [inline .github/workflows/rust.yml:183]: just a note: this toolchain is long deprecated...

File: .github/workflows/rust.yml (hunk 1 above)
Comment (Meister1593) [reply .github/workflows/rust.yml:195]: i've made x264 to be shipped with alvr with --gpl build

**Review by Meister1593 — APPROVED** [review]
Other than using deprecated actions-rs/toolchain it's fine

**Comment by zarik5** [comment]
I think the comments should be resolved in separate PRs

**Pull request merged**